desktop/core/src/desktop/models.py

```diff
@@ -1751,20 +1751,13 @@ def get_cluster_config(user):
 
 def get_remote_home_storage(user=None):
   remote_home_storage = REMOTE_STORAGE_HOME.get() if hasattr(REMOTE_STORAGE_HOME, 'get') and REMOTE_STORAGE_HOME.get() else None
-
-  if not remote_home_storage:
```

Comment:
is this condition not required any more?

Reply:
This condition was setting the value from core-site.xml for S3 browser which we don't want to depend on. 

Most of the time, the users set the REMOTE_STORAGE_HOME in the config so this does not gets executed + the S3 logic now lies in dedicated `get_s3_home_directory` method.